### PR TITLE
Add qpc server status command to CLI

### PR DESCRIPTION
**Pull request by abaiken**

### CODE REVIEW

File: qpc.spec [hunk 1]
```diff
@@ -56,6 +56,8 @@ install -D -p -m 644 build/qpc.1 $RPM_BUILD_ROOT%{_mandir}/man1/qpc.1
 %{_mandir}/man1/qpc.1.gz
 
 %changelog
+* Mon Oct 8 2018 Ashley Aiken <aaiken@redhat.com> 0.0.44-1
```
Comment (myersCody) [inline qpc.spec]: Needs the correct version

Comment (kholdaway) [inline qpc.spec]: Version `0.0.45-1`

File: qpc/messages.py [hunk 2]
```diff
@@ -244,6 +244,10 @@
     'The server will be contacted via "%s" at host "%s" with port "%s".'
 SERVER_INTERNAL_ERROR = 'An internal server error occurred. For more '\
     'information, see the server log file.'
+SERVER_STATUS_FAILURE = 'Unexpected failure occurred when accessing the '\
+    'status endpoint.'
+STATUS_PATH_HELP = 'Output file location.'
+STATUS_SUCCESSFULLY_WRITTEN = 'Server status written successfully..'
```
Comment (kholdaway) [inline qpc/messages.py]: Looks like there are periods `..`

File: qpc/server/commands.py [hunk 3]
```diff
@@ -15,3 +15,4 @@
 from qpc.server.configure_host import ConfigureHostCommand
 from qpc.server.login_host import LoginHostCommand
 from qpc.server.logout_host import LogoutHostCommand
```
Comment (kholdaway) [inline qpc/server/commands.py]: Copyright should be updated to have 2017-2018 date.

File: qpc/server/__init__.py [hunk 4]
```diff
@@ -15,6 +15,8 @@
 CONFIG = 'config'
 LOGIN = 'login'
 LOGOUT = 'logout'
+STATUS = 'status'
```
Comment (kholdaway) [inline qpc/server/__init__.py]: Copyright should be updated to have 2017-2018 date.

File: quipucords/api/messages.py [hunk 5]
```diff
@@ -140,7 +140,7 @@
 # Scan Manager/Signal
 SIGNAL_STATE_CHANGE = 'SIGNAL %s received for scan job.'
 SIGNAL_SCAN_MANAGER_CRASH = 'Process unexpectedly crashed.  See logs.'
-SIGNAL_SCAN_MANAGER_RESTART = 'Recovering manager..'
+SIGNAL_SCAN_MANAGER_RESTART = 'Recovering manager.'
```
Comment (kholdaway) [inline quipucords/api/messages.py]: Copyright should be updated to have 2017-2018 date.

Comment (abaiken) [reply quipucords/api/messages.py]: fixed 👍

File: qpc/server/__init__.py (hunk 4 above)
Comment (abaiken) [reply qpc/server/__init__.py]: fixed 👍

File: qpc/server/commands.py (hunk 3 above)
Comment (abaiken) [reply qpc/server/commands.py]: fixed 👍

**Pull request merged**